src/rendertron.ts

```diff
@@ -149,14 +152,15 @@ export class Rendertron {
   async handleHealthRequest(ctx: Koa.Context) {
     const { concurrency, count } = counter.getCounts()
 
-    if (concurrency <= concurrencyKillCount) {
-      logger.info(`Health check passed <= ${concurrencyKillCount}`, { render_concurrency: concurrency, render_count: count })
+    if (concurrency <= concurrencyKillCount && cooldownCount === 0) {
```

Comment:
`cooldownCount` starts at 5, so we will start off by reporting the pod as unhealthy 😅 
Do we also want to reset the cooldown? When it becomes healthy again.

```suggestion
    const isOnCooldown = cooldownCount < 5 && cooldownCount != 0;
    if (concurrency <= concurrencyKillCount && !isOnCooldown) {
       resetCooldown();
       ...
```

Reply:
D'oh - fixed and also showed via logs what is now going on ....